pkg/reconciler/nodeconfig_reconciler.go

```diff
@@ -27,14 +28,18 @@ const (
 	StatusProvisioned  = "provisioned"
 
 	controlPlaneLabel = "node-role.kubernetes.io/control-plane"
+	numOfRefs         = 2
+	configThreshold   = time.Minute * 2
```

Comment:
`configTimeout` is more accurate for this one imo

Reply:
Done